pontoon/api/schema.py

```diff
@@ -112,6 +115,54 @@ def resolve_localizations(obj, _info, include_disabled):
         return qs.filter(project__disabled=False)
 
 
+class ActionLog(DjangoObjectType):
+    class Meta:
+        model = ActionLogModel
+        only_fields = (
+            "action_type",
+            "created_at",
+            "performed_by",
+            "translation",
+        )
+
+
+class User(DjangoObjectType):
+    class Meta:
+        convert_choices_to_enum = False
+        model = UserModel
+        only_fields = (
+            "username",
+            "first_name",
+            "last_name",
+        )
+
+    actions = graphene.List(ActionLog)
+
+    def resolve_actions(obj, info):
+        return ActionLogModel.objects.filter(performed_by=obj)
+
+
+class Translation(DjangoObjectType):
+    class Meta:
+        model = TranslationModel
+        only_fields = (
+            "user",
+            "locale",
+            "string",
+            "plural_form",
+            "date",
+            "active",
+            "fuzzy",
+            "approved",
+            "rejected",
```

Comment:
Until https://bugzilla.mozilla.org/show_bug.cgi?id=1179954 is fixed, it'd be nice to have a `status` property on the `Translation` model. We did something similar for `missing_strings` and `complete` and make use of that in the schema.